pyrallis/utils.py

```diff
@@ -572,6 +574,60 @@ def deflatten(d: Dict[str, Any], sep: str = '.'):
     return deflat_d
 
 
+def parse_string(s):
+    if Config.get_config_type() == ConfigType.YAML:
+        import yaml
+        return yaml.safe_load(s)
+    elif Config.get_config_type() == ConfigType.JSON:
+        import json
+        try:
+            return json.loads(s)
+        except json.decoder.JSONDecodeError:
+            return s # No parsing to be done, simply return value
+    elif Config.get_config_type() == ConfigType.TOML:
+        import toml
+        try:
+            return toml.loads(f'val = {s}')['val']
```

Comment:
Maybe extract this line to a function with an explanation comment in it?

Reply:
Yeah, could be easier to read